compile broken by protos

This happens on both Mac OS X and Linux with a clean checkout with `.gradlew check`:


```
> Task :grpc-compiler:compileJava_pluginExecutableJava_pluginCpp
In file included from /Users/elharo/grpc-java/compiler/src/java_plugin/cpp/java_plugin.cpp:8:
/Users/elharo/grpc-java/compiler/src/java_plugin/cpp/java_generator.h:8:10: fatal error: 'google/protobuf/io/zero_copy_stream.h' file not found
#include <google/protobuf/io/zero_copy_stream.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.

/Users/elharo/grpc-java/compiler/src/java_plugin/cpp/java_generator.h:8:10: fatal error: 'google/protobuf/io/zero_copy_stream.h' file not found
#include <google/protobuf/io/zero_copy_stream.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.

In file included from /Users/elharo/grpc-java/compiler/src/java_plugin/cpp/java_generator.cpp:1:
/Users/elharo/grpc-java/compiler/src/java_plugin/cpp/java_generator.h:8:10: fatal error: 'google/protobuf/io/zero_copy_stream.h' file not found
#include <google/protobuf/io/zero_copy_stream.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -9,6 +9,11 @@ If you are new to github, please start by reading [Pull Request howto](https://h
 In order to protect both you and ourselves, you will need to sign the
 [Contributor License Agreement](https://identity.linuxfoundation.org/projects/cncf).
 
+## Compiling
+
+See [COMPILING.md](COMPILING.md). Specifically, you'll generally want to set
+`skipCodegen=true` so you don't need to deal with the C++ compilation.
+
 ## Code style
 
 We follow the [Google Java Style

